plugins/notifications-backend/src/service/router.ts

```diff
@@ -204,6 +204,13 @@ export async function createRouter(
       opts.read = false;
       // or keep undefined
     }
+    if (req.query.created_after) {
+      const sinceEpoch = Date.parse(req.query.created_after.toString());
```

Comment:
```suggestion
      const sinceEpoch = Date.parse(String(req.query.created_after));
```

Ever so slightly more safer way to convert to a string, in case of `undefined` / `null`